packages/skeleton-svelte/src/lib/components/InputChips/InputChips.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts">
+	import * as tagsInput from '@zag-js/tags-input';
+	import { useMachine, normalizeProps } from '@zag-js/svelte';
+	import { useId } from '$lib/internal/use-id.js';
+	import type { TagsInputProps } from './types.js';
+
+	// Props
+	let {
+		value = $bindable(),
+		base = 'input',
+		inputBase = 'bg-transparent border-transparent focus:border-transparent focus:ring-0',
+		inputClasses = '',
+		chipContainerBase = 'px-3 py-2 flex gap-1',
+		chipContainerClasses = '',
+		chipBase = 'preset-filled-surface-950-50 px-2 py-1 rounded',
+		chipClasses = '',
+		editInputBase = 'p-0 bg-transparent leading-3 border-transparent focus:border-transparent focus:ring-0',
+		editInputClasses,
+		classes = '',
+		...zagProps
+	}: TagsInputProps = $props();
+
+	// Machine
+	const [snapshot, send] = useMachine(
+		tagsInput.machine({
+			id: useId(),
+			onValueChange: (details) => {
+				value = details.value;
+			}
+		}),
+		{
+			context: {
+				...zagProps,
+				get value() {
+					return $state.snapshot(value);
+				}
+			}
+		}
+	);
+
+	// API
+	const api = $derived(tagsInput.connect(snapshot, send, normalizeProps));
+</script>
+
+<div class="{base} {classes}" {...api.getRootProps()}>
+	<input class="{inputBase} {inputClasses}" {...api.getInputProps()} />
```

Comment:
We need a `placeholder` prop to apply here. It's ok to use English text as the default long as this can be customized. Let's avoid the use of "tags" or "chips" terms though. Maybe just `Add...` for now.

Reply:
Adding text feels a bit opinionated, I'd rather keep it empty (but definitely available)

Reply:
It needs to be added to the examples at least. Otherwise it may make folks think it doesn't support this.